src/AutoSaveStatus/useAutoSaveStatus.test.tsx

```diff
@@ -15,69 +18,47 @@ describe(useAutoSaveStatus, () => {
   });
 
   it("renders with a provider", () => {
-    const { result } = renderHook(() => useAutoSaveStatus(), {
-      wrapper: ({ children }) => <AutoSaveStatusProvider>{children}</AutoSaveStatusProvider>,
-    });
+    const { result } = renderHook(() => useAutoSaveStatus(), { wrapper });
 
     expect(result.current.status).toBe(AutoSaveStatus.IDLE);
   });
 
   it("indicates when something is in-flight", () => {
-    const { result } = renderHook(() => useAutoSaveStatus(), {
-      wrapper: ({ children }) => <AutoSaveStatusProvider>{children}</AutoSaveStatusProvider>,
-    });
+    const { result } = renderHook(() => useAutoSaveStatus(), { wrapper });
 
-    act(() => result.current.triggerAutoSave());
+    act(() => result.current.setLoading(true));
```

Comment:
Moving over to the `setLoading` API.

The context still has `trigger`/`resolve` as it's internal impl that we expose to `useAutoSaveStatus`, but the external API of `useAutoSaveStatus` is now just "loading yes/no?" and internally we turn that into trigger/resolve as/if needed.